apps/cnquery/cmd/sbom.go

```diff
@@ -27,6 +27,7 @@ func init() {
 	sbomCmd.Flags().String("asset-name", "", "User-override for the asset name")
 	sbomCmd.Flags().StringToString("annotation", nil, "Add an annotation to the asset.") // user-added, editable
 	sbomCmd.Flags().StringP("output", "o", "list", "Set output format: "+sbom.AllFormats())
+	sbomCmd.Flags().Bool("with-evidences", false, "Display evidence for each component")
```

Comment:
I'm not sure about "evidence**s**". @misterpantz Is this word still used?

Reply:
Checked Oxford dictionary:

```
In general English, evidence is always uncountable. However, in academic English the plural evidences is sometimes used: (specialist) The cave contained evidences of prehistoric settlement.
```

So technically it works but we should make this simpler and just use `with-evidence`

Reply:
Renamed to `with-evidence`